live/full/global.ini

```diff
@@ -40166,7 +40166,7 @@ RetrieveDatapad_obj_short_01=Steigen Sie an Bord der "Stolz von Arliss"
 RetrieveDatapad_obj_short_02=Schiffskapitän suchen
 RetrieveDatapad_obj_short_03=Erhalten Sie Zugang zur Leiche des Kapitäns
 RetrieveDatapad_obj_short_04=Durchsuchen Sie die Leiche des Kapitäns nach seinem Datapad
-RetrieveDatapad_obj_short_05=Daten senden
+RetrieveDatapad_obj_short_05=Daten übermitteln (Auftragsmanager)
```

Comment:
Ich denke das (Auftragsmanager) kann man weg lassen da der Button unten nun Übermitteln heißt.

Reply:
Man weiß aber trotzdem nicht wo/wie man es Übermitteln muss. Sicher dass ich das weg machen soll?

Reply:
Also ich denke das es jetzt offensichtlicher ist als vorher. Vorher stand ja „Daten senden“ und unten „Abschließen“. 

Jetzt steht in der Quest „Daten übermitteln“ und unten „Übermitteln“. 

Also ich denke das es jetzt offensichtlicher ist. Eigentlich ist es ja CIG Aufgabe. Die könnten den Button unten auch grün einfärben 😁